Dockerfile

```diff
@@ -3,8 +3,11 @@ FROM jberrenberg/quake3:latest
 LABEL maintainer "snapcase"
 
 RUN \
-  wget http://playmorepromode.org/CPMA148.zip -O /tmp/CPMA148.zip && \
-  unzip /tmp/CPMA148.zip -d ~/ioquake3 && \
-  rm /tmp/CPMA148.zip
+  wget https://playmorepromode.org/files/latest/cpma-1.50-nomaps.zip -O /tmp/cpma-1.50-nomaps.zip && \
+  unzip /tmp/cpma-1.50-nomaps.zip -d ~/ioquake3 && \
+  rm /tmp/cpma-1.50-nomaps.zip && \
+  wget https://playmorepromode.org/files/cpma-mappack-full.zip -O /tmp/cpma-mappack-full.zip  && \
+  unzip /tmp/cpma-mappack-full.zip  -d ~/ioquake3 && \
```

Comment:
this needs to be extracted to `~/ioquake3/baseq3`

Reply:
hm, thought I could point to a specific line. it's the zip with the mappack

Reply:
Changed that.